composer.json

```diff
@@ -18,7 +18,7 @@
         "zendframework/zend-validator": "^2.6"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.5",
+        "phpunit/phpunit": "^5.7.21 || ^6.3.0",
```

Comment:
It's not too important, but we don't need `.0` in `6.3.0`, please note we have it like that for other packages, for example:
`"zendframework/zend-validator": "^2.6"`
not:
`"zendframework/zend-validator": "^2.6.0"`